deploy/kubedirector/kubedirector_v1alpha1_kubedirectorcluster_crd.yaml

```diff
@@ -75,7 +84,7 @@ spec:
                         value:
                           type: string
                   storage:
-                    required: [size]
+                    # required: [size]
```

Comment:
Curious about this change. In cases where the storage property is populated with some object value, the size property in that object is indeed required.

Reply:
yes size is required

Reply:
The CR yaml files were created by the operator SDK. We don't really need them so I will delete them.

Reply:
I'm glad you raised the size issue. I had to temporarily comment out the requirement for the size parameter to make progress. I believe the intent is that if (and only if) storage is specified then the size is required. If storage is not specified then the size is not required. I was getting a validation error that size was required even though the KubeDirectorCluster I was trying to create had no storage. I could use some help in debugging this issue.